clusters/app.ci/prow/03_deployment/pj-rehearse-plugin.yaml

```diff
@@ -76,8 +76,8 @@ items:
             mountPath: /var/tmp
           resources:
             requests:
-              memory: "3Gi"
-              cpu: "750m"
+              memory: "12Gi"
```

Comment:
This sounds a lot.

Reply:
It was using about `7Gi` when I triggered rehearsals on 3 different PRs all at the same time. I just wanted to be safe with the `12Gi`. Perhaps we can go to 7 and see how it goes?